engine/release-notes.md

```diff
@@ -25,7 +25,25 @@ consistency and compatibility reasons.
 ## 18.09.1
 2019-01-09
 
-### Security fixes for Docker Engine EE and CE 
+#### Important notes about this release
+
+Docker Engine 18.09.0 and up install containerd as a separate systemd unit. Containerd
+therefore uses its own mount-namespace. The Docker Engine cannot run succesfully
```

Comment:
I think only it's own mount namespace in certain circumstances.

Reply:
But the the issues is we don't want them to be in separate namespaces with non-shared propagation.

Reply:
As long as it looks good to @thaJeztah I'm good with this.